### DISCUSSION TITLE
Make toggling Storyboard and video seperate per beatmap.

### DISCUSSION THREAD

**Discussion by TheFantasticLoki** [ppy/osu#17074] [Ideas] [answered]
The reason I propose this idea is purely because "most" storyboards cause far less lag than the avg video background. So being able to have storyboards on most maps and not have to worry about videos bogging down my fps would be nice. Like I can do in stable.

**Comment by frenzibyte** [comment, marked as answer]
This has been raised before (https://github.com/ppy/osu/discussions/13134). [Please search before opening discussions next time.](https://github.com/ppy/osu/discussions/16869)

**Reply by TheFantasticLoki** [reply]
I did do a search but nothing relevant showed in my results, sorry I'll try to do multiple searches next time but github search hardly ever gives me results I want.
